### ISSUE TITLE
Universal Rendering

### ISSUE THREAD

**Issue by serkandyck** [vmware/clarity#326]
trying add Clarity Ui for server rendering but i get error

```
C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:55915
        var /** @type {?} */ parentCtor = Object.getPrototypeOf(typeOrFunc.prototype).constructor;
                                                 ^

TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (native)
    at ReflectionCapabilities.annotations (C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:55915:50)
    at Reflector.annotations (C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:56107:44)
    at DirectiveResolver.isDirective (C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:15407:61)
    at CompileMetadataResolver._getTypeDescriptor (C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:16707:37)
    at C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:16591:130
    at Array.forEach (native)
    at CompileMetadataResolver.getNgModuleMetadata (C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:16576:49)
    at JitCompiler._loadModules (C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:48179:64)
    at JitCompiler._compileModuleAndComponents (C:\Users\BioFr\Desktop\Bahis\dist\server\index.js:48139:52)
```

**Comment by jeeyun** [comment]
Hi @SerkanDayicik, can you give us some more info? Is your app based on the starter app from Angular Universal's [quickstart page](https://universal.angular.io/quickstart/)? Giving us some more information on your setup and/or code snippet would help us to resolve issues in a more timely manner.

**Comment by SerkanDayicik** [comment]
hey @jeeyun i'm using this starter [https://github.com/pmachowski/angular2-starter-kit] and i try impot ClarityModule from browser.module.ts this not error but angular say <clr-icons not found if i try add ClarityModule from node.module.ts or directly app.module.ts i get this error i try both forRoot and forChild

**Comment by el-davo** [comment]
Came across this issue also. I am following the offical angular-cli guide - https://github.com/angular/angular-cli/wiki/stories-universal-rendering. This works with the offical angular materia https://material.angular.io/l. However doesnt seem to be supported yet in clarity.

**Comment by exequiel09** [comment]
Mine works using [Clarity Seed][clarity-seed] and the [Angular CLI Universal story integration][angular-cli-universal-instruction]. But this is only minimal using only header and the vertical nav components.

[clarity-seed]: https://github.com/vmware/clarity-seed
[angular-cli-universal-instruction]: https://github.com/angular/angular-cli/wiki/stories-universal-rendering

**Comment by gnomeontherun** [comment]
This should be resolved with the 0.11 release. Please open a new ticket with updated details since universal support has changed in Angular since this was opened.
